integration-cli/docker_api_stats_test.go

```diff
@@ -231,6 +231,9 @@ func (s *DockerSuite) TestApiStatsContainerNotFound(c *check.C) {
 func (s *DockerSuite) TestApiStatsContainerGetMemoryLimit(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 
+	if !isCgroupMounted("memory") {
```

Comment:
We should add some daemon requirement on these limits. It seems it could work as the `/info` endpoint returns `"KernelMemory": true` and `"MemoryLimit": true`, so we could check the daemon when/on running tests (but it's gonna be tricky to do – adding this to my TODO).